models/public/faceboxes/faceboxes.md

```diff
@@ -0,0 +1,80 @@
+# faceboxes
+
+## Use Case and High-Level Description
+
+FaceBoxes: A CPU Real-time Face Detector with High Accuracy. For details see the [repository](https://github.com/zisianw/FaceBoxes.PyTorch), [paper](https://arxiv.org/pdf/1708.05234.pdf)
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Object detection                         |
+| Source framework                | PyTorch\*                              |
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `input.1` , shape - [1x3x1024x1024], format [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order - BGR.
+   Mean values - [104.0, 117.0, 123.0]
+
+### Converted model
+
+The converted model has the same parameters as the original model.
+
+## Output
+
+### Original model
+
+Vectors of floating-point values - boxes,scores and ?
+ - boxes_out, name - 342, shape - [1x21824x4]
+ - scores_out, name - 353, shape - [1x21824x2]
+ - ?, name - 354, shape - [2x21824x4]
```

Comment:
it is not a part of the model, it is just constant for storing anchor values

Reply:
Fixed